style-guide/assets/wds-icons-tag.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="15" height="15" viewBox="0 0 15 15">
```

Comment:
All icons should be `width="18" height="18" viewBox="0 0 18 18"` to have a consistent size.